bundles/AdminBundle/Security/User/UserChecker.php

```diff
@@ -34,8 +34,6 @@ class UserChecker extends BaseUserChecker
     public function checkPreAuth(UserInterface $user)
     {
         $this->checkValidUser($user);
-
-        parent::checkPreAuth($user);
```

Comment:
Also replace the deprecated the parent class `UserChecker` with `InMemoryUserChecker`

Reply:
@dvesh3 I changed it https://github.com/pimcore/pimcore/pull/12193/commits/8bf8e81117dc1a5cab8829c0a1519468301f4daa The InMemoryUserChecker class have the methods. So I add it again.